Cms_frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+// import LoginForm from 'src/components/LoginScreen/LoginForm';
+import {Typography, Button} from '@material-ui/core';
+import 'src/css/index.css'
+
+const LoginScreen: React.FC = () => {
+	const adminUser = {
+		email: "admin@admin.com",
+		password: "admin123"
+	}
+
+    const [details, setDetails] = useState({email: "", password: ""});
+	const [email, setEmail] = useState({email: ""});
+	const [password, setPassword] = useState({password: ""});
+	const [error, setError] = useState("");
+
+	const Login = () => {
+		console.log(details);
+
+		if (details.email === adminUser.email && details.password === adminUser.password) {
+			console.log("Logged in");
+			setEmail({
+				email: details.email
+			})
+            setPassword({
+                password: details.password
+            })
+		} else {
+			console.log("Incorrect login details")
+			setError("Incorrect login detals")
+		}
+
+	}

```

Comment:
this isnt actually handling the login with the backend, so it can also be deleted :(